backend/src/trainees/trainees.controller.ts

```diff
@@ -59,16 +66,34 @@ export class TraineesController {
   @ApiResponse({
     type: TraineeAddNewResponseDto,
   })
-  @ApiBody({ type: TraineeAddNewRequestDto })
-  @Post()
-  async create(
-    @Body() { trainee }: TraineeAddNewRequestDto,
-    @Request() req: ExpressRequest,
+  @ApiBody({ type: TraineeAddNewRequestSignupDto })
+  @Post('/signup')
```

Comment:
This endpoint should be available to the public, shouldn't it?

Reply:
I'm not sure I understand what you are saying right now. Could you please rephrase it @bibixx ?

Reply:
Well, right now you have to be logged in as an Admin or Instructor to access this endpoint

Reply:
@bibixx Please double check if the changes are ok right now.